### PR TITLE
Blog post for Infection 0.14.0

### PR DESCRIPTION
**Pull request by maks-rafalko**
There were so many commits and work done for this release, but I have difficulties writing this post as there are really not many features for "the end user".

Feel free to suggest, commit directly to this branch, comment. I'm going to publish and tag `0.14.0` this weekend.

### CODE REVIEW

File: src/_posts/whats-new-in-0.14.0.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+Infection now requires PHP 7.2.9+. If you can't for some reason upgrade, don't worry, you can still use previous versions.
+
+Most of the work in `0.14.0` was done to improve stability, performance and DX. However, there are a couple of new features and enhancements.
+
+### Performance and memory consumption
+
+We [made an update](https://github.com/infection/infection/pull/710) to dramatically reduce memory usage by using classes instead of object-like arrays. For example, running Infection for Psalm now takes 58% less memory (`5G` instead of `12G`).
+
+> Read the blog post https://steemit.com/php/@crell/php-use-associative-arrays-basically-never to understand why you should prefer classes
```
Comment (sanmai) [inline src/_posts/whats-new-in-0.14.0.md]:
```suggestion
> Read [this blog post](https://steemit.com/php/@crell/php-use-associative-arrays-basically-never) to understand why you should prefer classes.
```

File: src/_posts/whats-new-in-0.14.0.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+### Codeception
+
+We are working really hard to integrate Codeception into Infection. There were major blocker issues and we had to patch Codeception as well to fix them. We really hope to complete the work by `0.15.0` release in the next couple of months.
+
+### Split Infection to separate packages
+
+Currently, Infection is a quite a big project with all the code placed inside one repository. We will definitely split it to several ones.
+
+1. First of all, we want to extract all Mutators to a separate package. Read the explanation here https://github.com/infection/infection/issues/669
```
Comment (sanmai) [inline src/_posts/whats-new-in-0.14.0.md]:
```suggestion
1. First of all, we want to extract all Mutators to a separate package. [Read the reasoning here.](https://github.com/infection/infection/issues/669)
```

**Comment by sanmai** [comment]
Really looking towards the new release!

**Comment by maks-rafalko** [comment]
Thanks @sanmai

**Pull request merged**